### DISCUSSION TITLE
Code Block + Annotate (Copy only the code block to the clipboard)

### DISCUSSION THREAD

**Discussion by diogomdp** [GitbookIO/community#117] [Feature Requests]
### What's your feature/idea?

When clicking on the "Copy" button, the content of the `Code Block` and the `Annotations` are copied to the clipboard. My suggestion would be to only copy the content of the `Code Block` and ignore the `Annotations`.

## Content copied by `Gitbook`
![content copied by Gitbook](https://user-images.githubusercontent.com/52360869/221375179-00e22954-7dd1-4ead-aa8c-a9a941c4e784.png)

## Content copied by `Github` (desired outcome)
![content copied by Github](https://user-images.githubusercontent.com/52360869/221375233-efe8c852-3271-4dd9-aa42-7efb5fb9346a.png)

### What's the impact of this for you?

- [X] Critical
- [ ] High
- [ ] Medium
- [ ] Low

**Comment by taszap** [comment]
Hey, @astindev thank you for highlighting this! 
We are working on a fix, it should be shipped in our next release

**Reply by taszap** [reply]
Just to confirm this has now been shipped, thank you for your feedback.

**Reply by diogomdp** [reply]
Thank you very much!
